README.md

```diff
@@ -43,14 +48,24 @@ bash run_pipeline.sh
 
 For exact instructions on how to reproduce our analysis see `run_pipeline.sh`.
 
+## Results
+
+Our _in silico_ deconvolution of CD4+ cells, CD8+ cells, and Macrophages in TCGA
+data matches very closely to immunohistochemistry estimates of the same cell types
+in a separate dataset. The proportions of immune cell infiltrate across subtypes
+corresponds strongly.
+
+![immune deconvolution and IHC](figures/boxplot_validation_TCGA_summary.png?raw=true)
+
+We also observed that high macrophage infiltration was associated with worse outcomes
+in the TCGA dataset. This relationship was strengthed after adjusting for several
+covariates including age, gender, and gene expression based subtype.
+
+![TCGA macrophage survival](figures/TCGA_kaplanmeier_Macrophages.png?raw=true)  
+
+
 ## Contact
 
 For all code related questions, bug reporting, or feature requests please file a
 [GitHub issue](https://github.com/greenelab/GBM_immune_profiles/issues "file an issue")
 
-## Dependencies
-
-All code was performed in R version 3.2.3 and packages were managed by the
-[checkpoint](https://cran.r-project.org/web/packages/checkpoint/index.html "Microsoft checkpoint r package")
```

Comment:
> I removed this part of the README because I thought it was redudant.

Agree regarding redundancy. See my line comment.

> I could see it breaking if the base `ubuntu:16:04` version downloads a different R version for whatever reason

This shouldn't happen. From [here](https://help.ubuntu.com/community/UbuntuBackports):

> When Ubuntu releases a new version of its OS every 6 months, that release is largely frozen in time. While the software that is part of that release will get bug fixes and security patches, new major releases of software and the new features that come with them will not be available. 

So I think it's fine to not specify your R version... not sure if specifying a version is even possible if you're using the default ubuntu software channel.